### PR TITLE
OFMCC-4959 - Login updates

### PR DESCRIPTION
**Pull request by weskubo-cgi**
Simplified layout and updated login text.

### CODE REVIEW

File: frontend/src/views/LoginView.vue [hunk 1]
```diff
@@ -1,55 +1,57 @@
...
+            <br />
+            You must register for a Business BCeID before you can log in - it only takes a few minutes.
+          </v-card-text>
+          <v-card-actions>
+            <AppButton icon="mdi-login" href="//www.bceid.ca/register/business/getting_started/getting_started.aspx" @click="clearStorage" width="300px">Register for a BCeID</AppButton>
+          </v-card-actions>
+        </v-card>
+      </v-col>
+    </v-row>
+    <!-- </v-container> -->
```
Comment (vietle-cgi) [inline frontend/src/views/LoginView.vue:54]: should we remove this commented line?

File: frontend/src/views/LoginView.vue [hunk 2]
```diff
@@ -1,55 +1,57 @@
...
+    </v-row>
+    <v-row>
+      <v-col cols="12" md="6">
+        <v-divider />
+        <v-card flat>
+          <v-card-title>
+            <h4>Childcare Provider</h4>
+          </v-card-title>
+          <v-card-text>Log in with your primary Business BCeID</v-card-text>
+          <!-- <v-spacer></v-spacer> -->
```
Comment (vietle-cgi) [inline frontend/src/views/LoginView.vue:26]: should we remove this commented line?

**Review by vietle-cgi — CHANGES_REQUESTED** [review]

File: frontend/src/views/LoginView.vue (hunk 1 above)
Comment (weskubo-cgi) [reply frontend/src/views/LoginView.vue]: Yeah. Good catch.

File: frontend/src/views/LoginView.vue (hunk 2 above)
Comment (weskubo-cgi) [reply frontend/src/views/LoginView.vue]: Yeah. Good catch.

**Review by vietle-cgi — APPROVED** [review]

**Pull request merged**